acme/acme/crypto_util.py

```diff
@@ -161,31 +160,23 @@ def _pyopenssl_cert_or_req_san(cert_or_req):
     :rtype: `list` of `unicode`
 
     """
-    # constants based on implementation of
-    # OpenSSL.crypto.X509Error._subjectAltNameString
-    parts_separator = ", "
+    # constants based on PyOpenSSL certificate/CSR text dump
```

Comment:
On second thought, that comment is inadequate. Just added a more verbose version to the top of the function.